cli/template/base/src/env/schema.mjs

```diff
@@ -12,7 +12,7 @@ export const serverSchema = z.object({
 /**
  * You can't destruct `process.env` as a regular object in the Next.js
  * middleware, so you have to do it manually here.
- * @type {{ [k in keyof z.infer<typeof serverSchema>]: z.infer<typeof serverSchema>[k] | undefined }}
+ * @type {{ [k in keyof z.input<typeof serverSchema>]: z.input<typeof serverSchema>[k] | undefined }}
```

Comment:
This will always be `string | undefined`

```suggestion
* @type {{ [k in keyof z.input<typeof serverSchema>]: string | undefined }}
```

Reply:
Correct. Just fixed it everywhere